.github/workflows/ci.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       run: make validate-no-uncommitted-package-lock-changes
     - name: Install dependencies
       run: npm ci
+    - name: Check types
+      run: npm run check-types
```

Comment:
[inform] There were existing TypeScript errors in a couple components, not previously caught due to never failing CI when there are TypeScript errors.